macros/anaUltraThreshold.py

```diff
@@ -27,23 +32,29 @@
 print filename
 outfilename = options.outfilename
 
+vfat_mask = options.vfatmask
+
 import ROOT as r
 r.gROOT.SetBatch(True)
 GEBtype = options.GEBtype
 inF = r.TFile(filename+'.root')
 outF = r.TFile(filename+'/'+outfilename, 'recreate')
 
-VT1_MAX = 250
+#VT1_MAX = 250
```

Comment:
In the analysis routine I don't see the problem with drawing out to VT1=255.

But yes was informed by cbravo that in the scan application there is a firmware bug that prevents going out to full range of this register.